pkg/duck/subscriber_test.go

```diff
@@ -105,15 +106,15 @@ func TestSubscriberSpec(t *testing.T) {
 		},
 		"DNS Name": {
 			Sub: &v1alpha1.SubscriberSpec{
-				DeprecatedDNSName: &uri,
+				DeprecatedDNSName: &urlStr,
```

Comment:
i cross checked it with previous change, DeprecatedDNSName was testing string value "http://example.com". I'm not sure what's a valid behaviour over here.

Reply:
That's exciting.

I think this was probably a bug in the test, but since this is deprecated code and it looks like the current behavior will still work, this is probably acceptable.